sendwithus/__init__.py

```diff
@@ -546,13 +546,17 @@ def send(
             file_list = []
             if isinstance(files, list):
                 for f in files:
-                    file_list.append({
-                        'id': f.name,
-                        'data': (
-                            base64.b64encode(f.read()).decode()
-                            if six.PY3 else base64.b64encode(f.read())
-                        )
-                    })
+                    if isinstance(f, tuple):
```

Comment:
You mean to better do: 
"""[{'arbitrary_file_name.xyz': open('/home/Matt/report1.txt', 'r')}]"""?